src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.cs

```diff
@@ -162,5 +163,130 @@ internal static ulong ConvertAllAsciiBytesInUInt64ToLowercase(ulong value)
 
             return value ^ mask; // bit flip uppercase letters [A-Z] => [a-z]
         }
+
+        /// <summary>
```

Comment:
Can / should we be using any of these helpers from Ascii.EqualsIgnoreCase?

Reply:
Potentially, but I feel like that can/should be done in a follow up PR. I will log an issue.